benefits/core/admin.py

```diff
@@ -17,21 +17,69 @@
 GOOGLE_USER_INFO_URL = "https://www.googleapis.com/oauth2/v3/userinfo"
 STAFF_GROUP_NAME = "Cal-ITP"
 
+logger.debug("Register models with admin site")
+admin.site.register(models.PemData)
 
-for model in [
-    models.AuthProvider,
-    models.EligibilityType,
-    models.PaymentProcessor,
-    models.PemData,
-    models.TransitAgency,
-]:
-    logger.debug(f"Register {model.__name__}")
-    admin.site.register(model)
+
+@admin.register(models.AuthProvider)
+class AuthProviderAdmin(admin.ModelAdmin):  # pragma: no cover
+    def get_exclude(self, request, obj=None):
+        if not request.user.is_superuser:
+            return ["client_id_secret_name"]
+        else:
+            return super().get_exclude(request, obj)
+
+
+@admin.register(models.EligibilityType)
+class EligibilityTypeAdmin(admin.ModelAdmin):  # pragma: no cover
+    def get_exclude(self, request, obj=None):
+        if not request.user.is_superuser:
+            return []
+        else:
+            return super().get_exclude(request, obj)
 
 
 @admin.register(models.EligibilityVerifier)
-class SortableEligibilityVerifierAdmin(SortableAdminMixin, admin.ModelAdmin):
-    pass
+class SortableEligibilityVerifierAdmin(SortableAdminMixin, admin.ModelAdmin):  # pragma: no cover
+    def get_exclude(self, request, obj=None):
+        if not request.user.is_superuser:
+            return [
+                "api_auth_header",
+                "api_auth_key_secret_name",
+                "public_key",
+                "jwe_cek_enc",
+                "jwe_encryption_alg",
+                "jws_signing_alg",
+                "form_class",
```

Comment:
NB: this field is marked as `read` for Cal-ITP in the spreadsheet, but I think it makes sense to hide such a code-specific config from them.

Reply:
I can update the spreadsheet to reflect this

Reply:
Updated it

![image](https://github.com/user-attachments/assets/149fd895-0550-444d-a7c9-1e5019c834e6)